spanner/benchmark/bin/ycsb

```diff
@@ -16,5 +16,3 @@
 # A YCSB-like executable that can be integrated into PerfKitBenchmarker.
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-
-python ${DIR}/../ycsb.py "${@:1}"
```

Comment:
This PR should *not* touch files inside `spanner/` at all:  the previous version (before 96257f8) had stripped the trailing newline from that line.  *Removing* that line altogether is much worse.